src/Inputs/Slider.tsx

```diff
@@ -10,6 +10,8 @@ import styled, { DefaultTheme } from 'styled-components';
 import { LabelLayout, LabelLayoutProps } from '@Layouts';
 import { Popup } from '../Containers/Popup';
 
+let draggingThumb = 'Finish';
```

Comment:
still incorrect.

Reply:
I can use usestate for this then

Reply:
But why would it be incorrect now that it does not update with every render?

Reply:
Let's jump on a call afterwards.